app/graphql/mutations/add_daily_stat.rb

```diff
@@ -0,0 +1,37 @@
+module Mutations
+  class AddDailyStat < BaseMutation
+    argument :date, String, required: true
+    argument :data, GraphQL::Types::JSON, required: true
+
+    field :daily_stat, Types::DailyStatType, null: false
+    field :errors, [String], null: false
+
+    def resolve(date:, data:)
```

Comment:
this is breaking the build - too much complexity in this single method https://app.circleci.com/pipelines/github/failure-driven/health-tracker/33/workflows/dcf7cd8d-bf92-4b07-a918-ac3fa6ddcb7b/jobs/35

Reply:
function is lot smaller now, after next push build shouldn't fail